scalafix/rules/src/main/scala/fix/ClientFetchPatches.scala

```diff
@@ -1,45 +1,46 @@
+/*
+ * Copyright 2013-2020 http4s.org
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
 package fix
 
 import scalafix.v1._
 
 import scala.meta._
 
 object ClientFetchPatches {
-  def apply(tree: Tree)(implicit doc: SemanticDocument): List[Patch] = {
+  def apply(tree: Tree)(implicit doc: SemanticDocument): List[Patch] =
```

Comment:
All the changes here are just scalafmt.

Reply:
Yes, I totally fine with it here. But I would hesitate from applying scalafmt to input and output sources for scalafix. I described above why.

Reply:
Looks much better after scalafmt, btw.

Reply:
It does.  I reverted it to your last version, just to get something working.  We could tweak it, but like you say, it's the automation that's bad here.